### ISSUE TITLE
RELEASE 1.117.0

### ISSUE THREAD

**Issue by jlyon** [proudcity/wp-proudcity#1915]
### Changelog

New Features
- There are no new features in this release.

Bugfixes:
- Accessibility: Service center search button doesn't have accessible name or function #1896
- Accessibility: Remove 'Find question' nav in service center #1897

Hotfixes:
- Hotfix 1.116.1: Bump version on wp-proud-theme (with release)
- Hotfix 1.116.2: Fixing 'Recoverable fatal error: Object of class stdClass could not be converted to string' (wp-proud-search-elastic)

WordPress Core Update 
- The [WordPress Core 5.8.2 security and maintenance update](https://wordpress.org/news/2021/11/wordpress-5-8-2-security-and-maintenance-release/) is included in this release.

3rd party plugin updates (https://github.com/proudcity/wp-proudcity/commit/243eb8979d74a5888f64bbd6047ff68ad40004e9)
- disable-comments  | 2.2.3     | 2.2.4
- siteorigin-panels | 2.15.0    | 2.15.3 
- popup-maker       | 1.16.2    | 1.16.3 
- post-expirator    | 2.6.2     | 2.7.0   
- so-widgets-bundle | 1.25.1    | 1.26.0  
- w3-total-cache    | 2.1.9     | 2.2.0     
- wp-mail-smtp      | 3.1.0     | 3.2.1 
- wp-sitemap-page   | 1.9.0     | 1.9.1   
- duplicate-post    | 4.1.2     | 4.2      
- wordpress-seo     | 17.5      | 17.7.1       

## Scripted updates
```
# update wp-stateless setting
```
### Manual updates
- [ ] 
  ```
  ```
- [ ] Make prod test charge(s)

### Testing emphasis
- [ ] Check document uploads
- [ ] wp-fullcalendar
- [ ] Williams County: calendar category overlap #1798
- [ ] check that stripe feed add works
- [ ] login from my.proudcity.com
- [ ] login redirect
- [ ] Login
- [ ] Search

**Comment by jlyon** [comment]
Pushed, posted and scheduled to send @kevindherman @lukefretwell @aschmoe
